.mergify.yml

```diff
@@ -0,0 +1,11 @@
+pull_request_rules:
```

Comment:
I don't think we have mergify in the context of the HashiCorp Github Organization. Do you have an update on this @danieldreier ?

Reply:
Ok, since I suspect this might take a while, let's just drop the mergify config for now. We'll merge these dependabot PRs by hand.